### DISCUSSION TITLE
Does the report or article come with a title?

### DISCUSSION THREAD

**Discussion by scottiesan** [assafelovic/gpt-researcher#834] [Q&A]
How do you get the title from the report or article?

Thanks!
Scott

**Comment by ElishaKay** [comment]
Sup @scottiesan 

For which report_type?

I believe multi_agents should return an object, so maybe it will be there.

Check the fields in multi_agents/task.json

Print the response in backend/server.py or backend/websocket_manager.py

May the force be with you.
